subsys/bluetooth/services/Kconfig.ias

```
@@ -0,0 +1,35 @@
+# Bluetooth GATT Immediate Alert Service
+
+# Copyright (c) 2022 Codecoup
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BT_IAS
+	bool "Enable GATT Immediate Alert Service"
+
+if BT_IAS
+
+choice BT_IAS_DEFAULT_PERM
+	prompt "Default permissions used for IAS characteristics"
+	default BT_IAS_DEFAULT_PERM_W
+	help
+	  Default permissions for IAS characteristic attributes
+	  used when no permission is set for the report.
+	  Used also for boot reports.
+
+config BT_IAS_DEFAULT_PERM_W
+	bool "Write allowed"
+
+config BT_IAS_DEFAULT_PERM_W_ENCRYPT
+	bool "Require encryption for access"
+
+config BT_IAS_DEFAULT_PERM_W_AUTHEN
```

```suggestion
	default BT_IAS_DEFAULT_PERM
	help
	  Default permissions for IAS characteristic attributes
	  used when no permission is set for the report.
	  Used also for boot reports.

config BT_IAS_DEFAULT_PERM
	bool "Write allowed"

config BT_IAS_DEFAULT_PERM_ENCRYPT
	bool "Require encryption for access"

config BT_IAS_DEFAULT_PERM_AUTHEN
```

I think the `W` can be omitted since "write" is the only option to change.

---

Done